soc/PackEfficiency.py

```diff
@@ -0,0 +1,42 @@
+#this is programmed in notepad++ and not tested.
+
+import cmath
+
+class PackEfficiency:
+	'''
+	resistance of the pack in ohms
+	cell IR = 50mOhm
+	36 cells in parallel = 50mOhm / 36 = 1.389mOhm
+	36 modules in series = 1.389mOhm * 36 = 50mOhm
+	Contactor x4 = 0.8mOhm
+	Copper busbars, cables, and connections ~= 1mOhm
+	Fuse Resistance ~= 1mOhm
+	Connector resistance ~= 1mOhm
+	Total = ~55mOhm
+	
+	assume this is constant for now, implement changes for temp, SoC variations later
+	power loss in a pack is due primarily to ohmic heating
+	'''
+	
+	
+	def __init__(self):
+		_pack_resistance = 0.055
```

Comment:
This should be declared as self._pack_resistance, also any reason for _pack_resistance instead of pack_resistance?

Reply:
Copying Karl's style from his proposed BMS SoC algorithm, that's it.
I will try to do some testing before Saturday.